src/mqtt_schemas.rs

```diff
@@ -0,0 +1,114 @@
+

```

Comment:
the file should be named after the struct in it, or the most important one.

Reply:
yes, wasn't happy with the naming either, but couldn't think of anything else. Feel free to suggest an alternative or directly rename it to something else